pom.xml

```diff
@@ -18,9 +18,9 @@
         <version.debezium>${project.version}</version.debezium>
 
         <!-- Dependencies -->
-        <version.vitess.grpc>7.0.0</version.vitess.grpc>
+        <version.vitess.grpc>9.0.0</version.vitess.grpc>
```

Comment:
I'll update the compatibility matrix on the Debezium website as well

Reply:
Corresponding compatibility matrix PR: https://github.com/debezium/debezium.github.io/pull/641